bin/wasm-node/rust/src/ffi/bindings.rs

```diff
@@ -241,10 +241,19 @@ pub extern "C" fn init(
 /// The buffer passed as parameter **must** have been allocated with [`alloc`]. It is freed when
 /// this function is called.
 ///
+/// Additionally, an index that represents the source of the request is also passed as a parameter.
```

Comment:
Please mention that this `source_id` is completely opaque and can be chosen freely